app/src/main/res/layout/image_region_selection_interaction_item.xml

```diff
@@ -52,6 +52,7 @@
       app:entityId="@{viewModel.entityId}"
       app:imageUrl="@{viewModel.imagePath}"
       app:onRegionClicked="@{(region) -> viewModel.onClickableAreaTouched(region)}"
+      app:clickedRegions="@{viewModel.clickedRegions}"
```

Comment:
For this & selection interaction, why do we need these new bindings?

Reply:
Removed all newly created bindings.